test/test_playbooks/product.yml

```diff
@@ -1,22 +1,46 @@
 ---
-- hosts: localhost
+- hosts: fixtures
+  gather_facts: false
   tasks:
   - name: 'Load server config'
     include_vars:
       file: server_vars.yml
   - include: tasks/organization.yml
     vars:
       organization_state: present
+- hosts: tests
+  gather_facts: false
+  tasks:
+  - name: 'Load server config'
+    include_vars:
+      file: server_vars.yml
+  - include: tasks/product.yml
+    vars:
+      product_state: present
+      expected_change: true
   - include: tasks/product.yml
     vars:
       product_state: present
+      expected_change: false
   - include: tasks/product.yml
     vars:
       product_description: 'A product with an updated description'
       product_state: present
+      expected_change: true
   - include: tasks/product.yml
     vars:
       product_state: absent
+      expected_change: true
+  - include: tasks/product.yml
+    vars:
+      product_state: absent
+      expected_change: false
+- hosts: fixtures
```

Comment:
Could you just explain the fixtures vs. tests to me? I get that fixtures are setting up data for the tests, but is the purpose of setting the host like this for reusing fixture cassettes?

Reply:
The fixtures are being run without recording (see inventory).
In "normal mode" the tests are run alone from tape.
For "recording mode", the fixtures prepare the environment. And tear it down. And the tests run with recording against a real server.
Maybe we should include an additional mode that runs the whole playbook without recording against a real server...

Reply:
Ah thanks, that makes sense. I'd be fine without the additional mode for this PR. It seems like something that'd be easy to add later.

Reply:
We need a githubfeature: Open a this issue automatically when this PR is merged.